scripts/install-worker.sh

```diff
@@ -316,7 +316,7 @@ fi
 ################################################################################
 ### ECR CREDENTIAL PROVIDER ####################################################
 ################################################################################
-if [[ ! $KUBERNETES_VERSION =~ "1.19"* && ! $KUBERNETES_VERSION =~ "1.20"* && ! $KUBERNETES_VERSION =~ "1.21"* ]]; then
+if [[ ! $KUBERNETES_VERSION == "1.19"* && ! $KUBERNETES_VERSION == "1.20"* && ! $KUBERNETES_VERSION == "1.21"* ]]; then
```

Comment:
Bit concerned that this wasn't working before; can you elaborate on this?

Reply:
Also can you review #1070 ? Would make this change moot

Reply:
When I built AMIs 1.22+, I found that the configuration files didn't exist. With some testing, I verified that this version check logic doesn't work at all.